### PR TITLE
Docs for the check checker

### PR DESCRIPTION
**Pull request by obycode**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -175,6 +174,151 @@ $ clarinet check
...
+```clarity
+(define-public (dangerous (amount uint))
+    (let ((sender tx-sender))
+        ;; #[allow(unchecked_data)]
+        (as-contract (stx-transfer? amount tx-sender sender))
+    )
+)
+```
+
+**`#[filter(var1, var2)]`**
```
Comment (lgalabru) [inline README.md:308]: What's your take on tagging this annotation as the **recommended** way to deal with warnings?

**Review by lgalabru — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (obycode) [reply README.md:308]: I like that. How about I add this line below the description?

*This is the safest and preferred way to silence warnings that you consider false positives.*

Comment (lgalabru) [reply README.md:318]: 👍

**Review by lgalabru — APPROVED** [review]
Thanks @obycode!

**Pull request merged**